packages/core-api/src/app/App.tsx

```diff
@@ -139,6 +140,42 @@ function useConfigLoader(
   return { api: configReader };
 }
 
+class AppContextImpl implements AppContext {
+  constructor(private readonly app: PrivateAppImpl) {}
+
+  getPlugins(): BackstagePlugin<any, any>[] {
+    // eslint-disable-next-line no-console
+    console.warn('appContext.getPlugins() is deprecated and will be removed');
```

Comment:
Maybe surround all these in a once() style thing to avoid excessive logging. Or not worth it?

Reply:
Really not expecting anyone to be calling this tbh, so not worth it :p